src/app/services/baw-api/audio-recording/audio-recordings.service.ts

```diff
@@ -53,6 +54,31 @@ export class AudioRecordingsService extends ReadonlyApi<AudioRecording> {
       this.filterByForeignKey(filters, "siteId", site) as Filters
     );
   }
+
+  /**
+   * Filter audio recordings by region id. For better performance
+   * provide the region model instead of id
+   *
+   * @param filters Audio recording filters
+   * @param region Region to filter by
+   */
+  public filterByRegion(
+    filters: Filters<IAudioRecording>,
+    region: IdOr<Region>
+  ) {
+    if (typeof region === "number") {
```

Comment:
Should be using a type guard here https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards